ui2/src/pages/login/Login.tsx

```diff
@@ -106,6 +112,26 @@ const LoginScreen: React.FC<Props> = ({ onLogin }) => {
   );
 };
 
+const QuickSetupButton = () => {
+  const [parties, setParties] = useState<PartyDetails[]>();
+  const history = useHistory();
+
+  useEffect(() => {
+    const parties = retrieveParties();
+    if (parties) {
+      setParties(parties);
+    }
+  }, []);
+
+  return (
+    <>
+      <Button className="ghost dark" onClick={() => history.push('/quick-setup')}>
+        Quick Setup
+      </Button>
+    </>
```

Comment:
redundant outer fragment `<> ... </>` can be removed

Reply:
also should probably be a `Link` or `NavLink` instead :D 

if you add the `'button ghost dark'` className, I _think_ it will be styled to look like a button...

Reply:
unfortunately that does not style - i think it could if we added some more foundational button styling but will save for different PR

Reply:
it is unfortunate due to the loss of the keyboard accessibility, but i do not think that it is urgent enough to block the rest of this PR. approved :D